FirebaseCrashlytics.podspec

```diff
@@ -45,7 +45,6 @@ Pod::Spec.new do |s|
   s.dependency 'FirebaseInstallations', '~> 1.1'
   s.dependency 'FirebaseAnalyticsInterop', '~> 1.2'
   s.dependency 'PromisesObjC', '~> 1.2'
-  s.dependency 'GoogleDataTransport', '~> 6.1'
```

Comment:
A `git grep` of Crashlytics shows deps on GoogleDataTransport but not GoogleDataTransportCCT:

$ git grep GoogleDataTransport
CHANGELOG.md: - [changed] Changed Crashlytics to use GoogleDataTransport to upload crashes (#4989)
Crashlytics/Controllers/FIRCLSReportUploader.h:#import <GoogleDataTransport/GDTCORTransport.h>
Crashlytics/Controllers/FIRCLSReportUploader.m:#import <GoogleDataTransport/GDTCOREvent.h>
Crashlytics/Controllers/FIRCLSReportUploader.m:#import <GoogleDataTransport/GDTCORTransport.h>
Crashlytics/FIRCrashlytics.m:#import <GoogleDataTransport/GDTCORTargets.h>
Crashlytics/FIRCrashlytics.m:#import <GoogleDataTransport/GDTCORTransport.h>
Crashlytics/Models/FIRCLSInternalReport.m:// TODO: Remove this class after the uploading of reports via GoogleDataTransport is no longer an
Crashlytics/Models/Record/FIRCLSReportAdapter.h:#import <GoogleDataTransport/GDTCOREventDataObject.h>
Crashlytics/Models/Record/FIRCLSReportAdapter.h:/// the information into the nanopb model to be used with GoogleDataTransport
UnitTests/FIRCLSReportAdapterTests.m:#import <GoogleDataTransport/GDTCOREvent.h>
UnitTests/FIRCLSReportAdapterTests.m:#import <GoogleDataTransport/GDTCORTargets.h>
UnitTests/FIRCLSReportAdapterTests.m:#import <GoogleDataTransport/GDTCORTransport.h>
UnitTests/Mocks/FIRMockGDTCoreTransport.h:#import <GoogleDataTransport/GDTCORTransport.h>